.gitignore

```diff
@@ -1,6 +0,0 @@
-﻿################################################################################
```

Comment:
we can keep this file because it prevents accidentally committing files like the intellisense database

Reply:
ok